datasets/verkiezingen/verkiezingen.json

```diff
@@ -0,0 +1,53 @@
+{
+  "type": "dataset",
+  "id": "verkiezingen",
+  "title": "Verkiezingen",
+  "status": "beschikbaar",
+  "description": "Gegevens over verkiezingen",
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "processenverbaal",
+      "title": "Processenverbaal",
+      "description": "URI en metadata processenverbaal in pdf per jaar, per stemlocatie",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["schema", "id"],
```

Comment:
Zou jaar ook niet verplicht moeten zijn, omdat het ook het display veld is?

Reply:
Ah goeie, bij nader inzien heb ik de ID er in gezet. Omdat deze bestaat uit het verkiezingsjaar en het volgnummer van het document. Dit zou uniek moeten zijn...